Clinical-Genomics/scout
# Fix missing clinsig info on variant page

Bug fix #1777

**How to test**:
1. On master branch on stage pick a case and filter for variants with a certain clinsig (i.e. pathogenic, conflicting pathogenicity interpret., ..)
1. Go to one of these variants and look at summary. Clinsig info should be missing.
1. Install this branch and note the difference.

**Expected outcome**:
The functionality should be working
Take a screenshot and attach or copy/paste the output.

**Review:**
